docs/docs/home/home.md

```diff
@@ -1,62 +1,124 @@
 ---
-title: Ready to embark on your Rill adventure? Let's dive in!
+title: Ready for Fast Dashboards with Rill?
 slug: /
-sidebar_label: Welcome to Rill
+sidebar_label: Home
 sidebar_position: 00
 hide_table_of_contents: true
 ---
 
-## Install Rill 
-Install Rill using the command below, then launch your first project to explore core features hands-on.
+import TileIcon from '@site/src/components/TileIcon';
+
+## Install 
+Install Rill using the command below, then launch your first project to explore core features hands-on. For more installation methods, see our [complete installation instructions docs](/home/install). 
 
 ```bash
 curl https://rill.sh | sh
 rill start my-rill-project
 ```
 
+
+## Quickstart
+
+Take a look at our [Quick start](get-started/get-started.md) for a quick guide to get started with your own data! We use a public GCS dataset but you can follow along with your own data. Take a look at our [connectors docs](/reference/connectors/) for the exact steps to connect to your data.
+
+Or, if you're looking for more guides, check out our [Guides](/guides/) section!
+
+
+
 <!-- <img src = 'https://storage.googleapis.com/prod-cdn.rilldata.com/docs/rill_hero.gif' class='rounded-gif' />
 <br /> -->
 
-## Quick Starts & Examples
+##  Examples
+
+Browse our [repository of examples](https://github.com/rilldata/rill-examples/) to find a project that suites your needs or see them in action in our demo project by hitting [Live Demo →](https://ui.rilldata.com/demo). Some projects have a walkthrough, too! 
+
+```bash
+git clone https://github.com/rilldata/rill-examples.git
+cd rill-examples/rill-openrtb-prog-ads #swap this to the project that interests you!
+curl https://rill.sh | sh
+rill start
+```
+
+### **Programmatic Ads/OpenRTB**: 
+
+Bidstream data for programmatic advertisers to optimize pricing strategies, look for inventory opportunities, and improve campaign performance
```

Comment:
Each of these descriptions could use a period at the end. (Like the kitchen-sink description.)